plugins/out_s3/s3.c

```diff
@@ -1447,6 +1526,18 @@ static struct flb_config_map config_map[] = {
      0, FLB_FALSE, 0,
     "Predefined Canned ACL policy for S3 objects."
     },
+    {
+     FLB_CONFIG_MAP_STR, "compression", NULL,
+     0, FLB_FALSE, 0,
+    "Compression type for S3 objects. 'gzip' is the only support value for now. "
```

Comment:
I don't like "for now" it doesn't sound like official language to me, sounds too conversational

"'gzip' is currently the only supported value"

Reply:
I see. Will change the word